src/templates/pages/index/data.json

```diff
@@ -3,5 +3,10 @@
     "title": "Product Name",
```

Comment:
For cards use one array with name and items inside it. 

"cards": [
   {
     "label": "",
     "state": "new",
     "price": "199",
     "name" : "Name product",
     "image" : "img/catalog-item-photo-1-2x.webp"
   },
   {
     "label": "Sold out",
     "state": "new",
     "price": "2999",
     "name" : "Name Product",
     "image" : "img/catalog-item-photo-1-2x.webp"
   }
}